proposals/incubation/kubeflow.md

```diff
@@ -0,0 +1,223 @@
+    * [Jagadeeshi2i](https://github.com/jagadeeshi2i) - Jagadeesh J (Ideas2IT Technologies)
+    * [james-jwu](https://github.com/james-jwu) - James Wu (Google)
+    * [Jeffwan](https://github.com/Jeffwan) - Jiaxin Shan (Bytedance)
+    * [ji-yaqi](https://github.com/ji-yaqi) - 
+    * [jlyaoyuli](https://github.com/jlyaoyuli) - Joe Li (Google)
+    * [JOCSTAA](https://github.com/JOCSTAA) - Josh Ogbonda (Google)
+    * [joeliedtke](https://github.com/joeliedtke) - Joe Liedtke (Google)
+    * [Johnugeorge](https://github.com/johnugeorge) - Johnu George (Nutanix)
+    * [Linchin](https://github.com/Linchin) - Lingqing Gan (Google)
+    * [Luotigerlsx](https://github.com/luotigerlsx) - Shixin 
+    * [Mameshini](https://github.com/mameshini) - Igor Mameshin
+    * [Mbaijal](https://github.com/mbaijal) - Meghna Baijal (Amazon)
+    * [Neuromage](https://github.com/neuromage) - Ajay Gopinathan (Google)
+    * [NikeNano](https://github.com/NikeNano) - Niklas Hansson (Northvolt)
+    * [Numerology](https://github.com/numerology) - Jiaxiao Zheng (ByteDance)
+    * [Ojarjur](https://github.com/ojarjur) - Omar Jarjur
+    * [PatrickXYS](https://github.com/PatrickXYS) - Yao Xiao (Google)
+    * [RedbackThomson](https://github.com/RedbackThomson) - Nicholas Thomson (Amazon)
+    * [Rileyjbauer](https://github.com/rileyjbauer) - Riley Bauer
+    * [rui5i](https://github.com/rui5i) - Rui Fang (Google)
+    * [sasha-gitg](https://github.com/sasha-gitg) - Sasha Sobran (Google)
+    * [Saurabh24292](https://github.com/saurabh24292) - 
+    * [Shrinath-suresh](https://github.com/shrinath-suresh) - 
+    * [SinaChavoshi](https://github.com/SinaChavoshi) - Sina Chavoshi (Google)
+    * [StefanoFioravanzo](https://github.com/StefanoFioravanzo) - StefanoFioravanzo (Arrikto)
+    * [Sudivate](https://github.com/sudivate) - Sushant Divate
+    * [Surajkota](https://github.com/surajkota) - Suraj Kota (Amazon)
+    * [Tomcli](https://github.com/Tomcli) - Tommy Li (IBM)
+    * [Yhwang](https://github.com/yhwang) - Yihong Wang (IBM)
+    * [Yilun-msft](https://github.com/yilun-msft) - 
+    * [zichuan-scott-xu](https://github.com/zichuan-scott-xu) - 
+    * [zijianjoy](https://github.com/zijianjoy) - James Liu (Google)
+    * [zpChris](https://github.com/zpChris) - 
+* Training-operator
+    * [Gaocegege](https://github.com/Gaocegege) - Ce Gao (TensorChord)
+    * [Jeffwan](https://github.com/Jeffwan) - Jiaxin Shan (Bytedance)
+    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
+    * [Terrytangyuan](https://github.com/Terrytangyuan) - Yuan Tang (Akuity)
+    * [zw0610](https://github.com/zw0610) - Wang Zhang (Tencent)
+* Katib
+    * [Andreyvelich](https://github.com/Andreyvelich) - Andrey Velichkevich (Apple)
+    * [Gaocegege](https://github.com/Gaocegege) - Ce Gao (TensorChord)
+    * [Hougangliu](https://github.com/Hougangliu) - Hougang Liu (IBM)
+    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
```

Comment:
@mchmarny, Could you also update this section like the following since we updated the OWNERS file recently?

https://github.com/kubeflow/katib/blob/f941ec61e5fe8aa17919c51a8b347914ae70b876/OWNERS

```md
    * [Andreyvelich](https://github.com/Andreyvelich) - Andrey Velichkevich (Apple)
    * [Gaocegege](https://github.com/Gaocegege) - Ce Gao (TensorChord)
    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
    * [tenzen-y](https://github.com/tenzen-y) - Yuki Iwai (CyberAgent)
```